ui/src/main.ts

```diff
@@ -8,7 +8,8 @@ import '@mdi/font/css/materialdesignicons.css'
 import TransformationModule from '@/transformation/module'
 import StorageModule from '@/storage/module'
 import AuthModule from '@/components/auth/module'
-import PipelineModule from '@/pipeline/module'
+import DatasourceModule from '@/pipeline/datasource.module'
+import PipelineModule from '@/pipeline/pipeline.module'
```

Comment:
Separate views for datasource and pipeline. Did not refactor some names in order to keep the PR smaller (I know it's already way too large)...